processes/releasing.md

```diff
@@ -0,0 +1,134 @@
+# Releasing
+
+This document details the high-level process of creating a Metal3 release, and
+the post-release actions required to add new release branches to the CI.
+
+Note the [TODO](#todo) section in the end.
+
+## Process
+
+High-level release process we followed in case it makes it clearer:
+
+1. [Release IPAM](https://github.com/metal3-io/ip-address-manager/blob/main/docs/releasing.md)
+1. [Release BMO](https://github.com/metal3-io/baremetal-operator/blob/main/docs/releasing.md)
+1. [Release CAPM3](https://github.com/metal3-io/cluster-api-provider-metal3/blob/main/docs/releasing.md)
+1. Update Metal3 dev-env and CI to support new releases
+
+## Update Metal3 dev-env and CI to support new releases
+
+If a new minor or major was released, we need to make changes in Metal3 dev-env
+and CI.
+
+### Tag Ironic and Mariadb images
+
+After releasing `v1.x.y` version of CAPM3, create an annotated tag with `capm3-`
+prefix + release version in
+[ironic-image](https://github.com/metal3-io/ironic-image) and
+[mariadb-image](https://github.com/metal3-io/mariadb-image) Github
+repositories.
+
+```bash
+git tag -s -a capm3-v1.x.y -m capm3-v1.x.y [optional sha, or tag^{}]
+git push origin capm3-v1.x.y
+```
+
+After this, [Ironic](https://quay.io/repository/metal3-io/ironic) and
+[MariaDB](https://quay.io/repository/metal3-io/mariadb) container images will
+be built automatically  with `capm3-v1.x.y` tag in Quay. Verify the build is
+triggered, as often Quay has disabled the build trigger due build failures or
+other reasons.
+
+### Create Jenkins jobs
+
+Two Jenkins jobs need to be created in
+[JJB](https://gerrit.nordix.org/plugins/gitiles/infra/cicd/+/refs/heads/master/jjb/metal3/):
+
+- a periodic job that runs on a regular basis.
+- a PR verification job that is triggered by a keyword on a PR targeted for that
+  release branch.
+
+[Gerrit PR](https://gerrit.nordix.org/c/infra/cicd/+/17709) that adds support
```

Comment:
Something like, "Prior art" (commonly used in CAPI for referencing example previous PR) may suit here and in all occurrences below (Example PR) better:
```suggestion
[Prior art](https://gerrit.nordix.org/c/infra/cicd/+/17709) that adds support
```

Reply:
Fixed